packages/vx-grid/src/grids/GridColumns.tsx

```diff
@@ -34,9 +29,8 @@ export default function Columns({
   offset,
   tickValues,
   ...restProps
-}) {
-  let ticks = scale.ticks ? scale.ticks(numTicks) : scale.domain();
-  if (tickValues) ticks = tickValues;
+}: AllProps<ScaleInput>) {
+  const ticks = tickValues || scale.ticks ? scale.ticks(numTicks) : scale.domain();
```

Comment:
I don't have a ton of context, but seems like you're changing the logic here and again further down? Does the TS migration mean we never want ticks = tickValues? It is safe to call `scale.ticks` if we're not sure it exists?

Reply:
I didn't really change the logic, just made it an immutable variable and simplified the expression. 

Same as before, if `tickValues` is set, it overrides everything. Otherwise we use `.ticks()` if it's defined and fallback to `.domain()` which is always defined.

Reply:
you were right, messed up the logic! fixed with some `()` 🙏